src/gui/rss/articlelistwidget.cpp

```diff
@@ -55,6 +55,7 @@ void ArticleListWidget::setRSSItem(RSS::Item *rssItem, bool unreadOnly)
 {
     // Clear the list first
     clear();
+    m_rssArticleToListItemMapping.clear();
```

Comment:
Ok. Then it should be added everywhere the list is modified (where we add/delete items), isn't it?